tensorflow_datasets/structured/movielens.py

```diff
@@ -48,22 +48,49 @@
 """
 
 _DESCRIPTION = """
-This dataset describes 5-star rating from MovieLens,
-a movie recommendation service.
-Users were selected at random for inclusion.
-There are 5 versions included: 25m, latest-small, 100k, 1m, 20m.
-In all datasets, the movies data and ratings data are joined on "movieId".
-The 25m dataset, latest-small dataset, and 1m dataset have the same schema
-and they do not include demographic data.
-The 25m dataset is the latest stable version of the MovieLens dataset.
-It is recommended for research purposes.
-The latest-small dataset is a small dataset with
-the same schema as 25m and 20m.
-It is updated over time by GroupLens and its checksum might need updating.
-The 20m dataset is one of the most used MovieLens datasets in academic papers
+This dataset contains a set of movie ratings from the MovieLens website, a movie
+recommendation service. This dataset was collected and maintained by [GroupLens]
+(https://grouplens.org/), a research group at the University of Minnesota. There
+are 5 versions included: "25m", "latest-small", "100k", "1m", "20m". In all
+datasets, the movies data and ratings data are joined on "movieId". The 25m
+dataset, latest-small dataset, and 20m dataset contain only movie data and
+rating data. The 1m dataset and 100k dataset contain demographic data in
+addition to movie and rating data.
+
+- "25m": This is the latest stable version of the MovieLens dataset. It is
+recommended for research purposes.
+- "latest-small": This is a small subset of the latest version of the MovieLens
+dataset. It is updated over time by GroupLens and its checksum might need
+updating.
+- "100k": This is the oldest version of the MovieLens datasets. It is a small
+dataset with demographic data.
+- "1m": This is the largest MovieLens dataset that contains demographic data.
+- "20m": This is one of the most used MovieLens datasets in academic papers
 along with the 1m dataset.
-The 100k dataset and 1m dataset are the oldest MovieLens datasets.
-They both contain demographic data and have the same schema.
+
+Feature Description:
+The features below are included in all versions.
+
+- "movie_id": a unique identifier of the rated movie
+- "movie_title": the title of the rated movie with the release year in
+parentheses
+- "movie_genres": a list of genres to which the rated movie belongs
+- "user_id": a unique identifier of the user who made the rating
+- "user_rating": the score of the rating on a five-star scale
+- "timestamp": the timestamp of the ratings, represented in seconds since
+midnight Coordinated Universal Time (UTC) of January 1, 1970
+
+The "100k" and "1m" version in addition includes the following demographic
+features.
+
+- "user_gender": gender of the user who made the rating; a true value
+corresponds to male
+- "user_age": age of the user who made the rating
+- "user_occupation_label": the occupation of the user who made the rating
```

Comment:
What's the difference between label and string? Are labels integer-encoded?

Reply:
Yes. Strings are the raw label used in the original datasets. The raw label is converted to a label from a standardized set and then encoded as integers to produce "user_occupation_label". Since the 100k dataset and the 1m dataset have different sets of occupation labels, the standardized "user_occupation_label" can allow users to transfer the model between the 100k dataset and the 1m dataset.

Reply:
Got it. Perhaps we can mention that use case in the documentation?

Reply:
Yes. I have added them in the documentation in 101221754d7b65b59f4bf4bcb2e45499572988c3.